### PR TITLE
fixes #124: Fix broken  form> select and form> Readonly normalized  component

### LINKED ISSUE #124: [BUG]Component form>select and form>Readonly normalizzato are broken
**Describe the bug**

Component form>select and form>Readonly normalizzato does not have proper alignment of it's elements
**To Reproduce**
1. Go to form component
2. Click on select and readonly normalizzato
3. See error

**Expected behavior**
Elements should have proper alignments like this
efer

**Screenshots**
Errors
sle

emai

**Other Required Information (please complete the following information):**
 - OS: [ubuntu]
 - Browser [chrome]

### PR DESCRIPTION
**Pull request by RMCoder198**
Fixes #124 

#### PR Checklist

- [x] My branch is up-to-date with the Upstream `next` branch.
- [ ] The unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:
component form> select and form> Readonly normalized  has been fixed
selel

wefew

### CODE REVIEW

File: stories/Form/Form.stories.js [hunk 1]
```diff
@@ -264,10 +264,10 @@ stories.add("Readonly", withDocs(Readonly, withInfo()(ReadonlyComponent)));
 const ReadonlyNormalizzatoComponent = () => (
     <Form className="m-3">
         <FormGroup>
+            <Label for="staticEmail" style={{position:'static'}}>Email</Label>        
             <Input plaintext readOnly id="staticEmail">
                 email@example.com
             </Input>
-            <Label for="staticEmail">Email</Label>
```
Comment (Diuda) [inline stories/Form/Form.stories.js]: I think the label needs to appear below the email id as far as design is concerned of `Form` components. Seems that way in all other places.

Comment (sanyamdogra) [reply stories/Form/Form.stories.js]: Hey @Diuda, It looks good to me. Have a  look at this [link](https://italia.github.io/bootstrap-italia/docs/form/select/). The labels are placed on the top.

Comment (Diuda) [reply stories/Form/Form.stories.js]: Just saw the link. I guess there's a conflict between the documentation on Bootstrap Italia page and how the labels have been structured in the current system then

Comment (sanyamdogra) [reply stories/Form/Form.stories.js]: Yes, that could be the case.

Comment (francescozaia) [reply stories/Form/Form.stories.js]: Yep, I confirm that. Things should be aligned to what you can see on Bootstrap Italia or on the [UI Kit](https://invis.io/VRR2TLB34TE) ([here](https://invis.io/D9R3VB52J8E#/291110316_It_Designers_UIkit_Components_Data_Entry_Input_Field) the page for input fields) that is the foundation for everything.

**Pull request closed without merging**